code/modules/power/cable.dm

```diff
@@ -415,3 +412,172 @@
 		logTheThing(LOG_STATION, user, "lays a cable[powered == 1 ? " (powered when connected)" : ""] at [log_loc(src)].")
 
 	return
+
+/// a cable spawner which can spawn multiple cables to connect to other cables around it.
+/obj/cablespawner
+	level = 1
```

Comment:
AFAIK level is irrelevant for things that shouldn't appear to players anyway.

Reply:
ah, didn't know this. Was just copying the `/obj/cable` code over. Will get rid of it